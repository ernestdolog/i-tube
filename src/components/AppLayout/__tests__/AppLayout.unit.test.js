import React from 'react';
import { shallow } from 'enzyme';
import { AppLayout } from '../AppLayout';

describe('AppLayout', () => {
  test('renders without props', () => {
    const wrapper = shallow(<AppLayout/>);
    expect(wrapper).toMatchSnapshot();
  });
});

test('renders <AppLayout/> with one child', () => {
    const wrapper = shallow(
      <AppLayout>
        <div>Child 1</div>
      </AppLayout>
    );
    expect(wrapper).toMatchSnapshot();
  });
  
  test('renders <AppLayout/> with two children', () => {
    const wrapper = shallow(
      <AppLayout>
        <div>Child 1</div>
        <div>
          <span>Child 2</span>
          <p>Child 2</p>
        </div>
      </AppLayout>
    );
    expect(wrapper).toMatchSnapshot();
  });