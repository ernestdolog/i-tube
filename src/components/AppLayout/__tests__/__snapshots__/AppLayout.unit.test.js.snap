// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout renders without props 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;

exports[`renders <AppLayout/> with one child 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child 1
  </div>
</div>
`;

exports[`renders <AppLayout/> with two children 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child 1
  </div>
  <div>
    <span>
      Child 2
    </span>
    <p>
      Child 2
    </p>
  </div>
</div>
`;
